Source-Swift/ACFloatingTextfield.swift

```diff
@@ -253,6 +253,7 @@ fileprivate extension ACFloatingTextfield {
         UIView.animate(withDuration: 0.2) {
             self.labelErrorPlaceholder?.alpha = 0;
             self.labelErrorPlaceholder?.frame = labelErrorFrame!;
+            self.bottomLineView?.backgroundColor = self.placeHolderColor;
```

Comment:
I think this is incorrect , why changing the line background color with placeholder color , it has the seperate property for this, i can merge this request but this needs to be removed (line no 256) Thanks